tsconfig.json

```diff
@@ -1,23 +1,20 @@
 {
     "compilerOptions": {
-        "target": "es2017",
+        "target": "ES2020",
```

Comment:
i agree with these new targets, but they should be updated within each pkg too

Reply:
I do remember updating it everywhere, maybe forgot to commit. Anyways should be updated now